projects_test.go

```diff
@@ -0,0 +1,51 @@
+package gitlab
+
+import (
+	"testing"
+)
+
+func TestListAllProjects(t *testing.T) {
+	ts, client := Stub("stubs/projects/index.json")
+	defer ts.Close()
+
+	opt := &ListProjectsOptions{}
+	projects, _, err := client.Projects.ListAllProjects(opt)
+
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	if len(projects) != 2 {
+		t.Fail()
+	}
+	if projects[0].Name != "project" {
+		t.Fail()
+	}
+	if projects[1].Name != "project2" {
+		t.Fail()
+	}
+}
+
+func TestGetProject(t *testing.T) {
+	ts, client := Stub("stubs/projects/show.json")
+	defer ts.Close()
+
+	project, _, err := client.Projects.GetProject(1)
+
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	if project.Name != "project" {
+		t.Fail()
+	}
+	if project.Namespace.Name != "group" {
+		t.Fail()
+	}
+	if project.Permissions.ProjectAccess.AccessLevel != MasterPermissions {
+		t.Fail()
+	}
+	if project.Permissions.GroupAccess != nil {
+		t.Fail()
+	}
+}
```

Comment:
Yes, that makes sense. I fixed it. 
I'm currently also looking at go-github client tests. 
https://github.com/google/go-github/blob/master/github/repos_test.go
And I like them :) Maybe we should implement tests the same way, but maybe not as part of this PR (?)
What do you think ?